Cloudinary/Operation/01_ManageCloudinaryFiles.md

```diff
@@ -0,0 +1,182 @@
+
+    > [Info] Currently only structured meta data fields of the text and the date field type are supported by Actindo. Note to enter a date value in the appropriate date format (yyyy-MM-dd).
+
+    ![Cloudinary image settings](../../Assets/Screenshots//PIM/Products/List/Attributes/CloudinaryImageSettings.png "[Cloudinary image settings]")
+
+8. Click the [SAVE] button in the upper right corner of the *Edit product* view.    
+  The imported images have been saved. The *Edit product* view is closed.
+
+
+
+## Remove a Cloudinary file assignment in PIM
+
+When removing an image assignment from Cloudinary in the *PIM* module, you can either keep the corresponding image in Cloudinary or delete the image in Cloudinary at the same time.
+
+#### Prerequisites
+
+At least one Cloudinary file has been assigned to the PIM product.
+
+#### Procedure
+
+*PIM > Products > Tab LIST*
+
+![Products](../../Assets/Screenshots/PIM/Products/List/Products.png "[Products]")
+
+> [Info] The following procedure describes exemplary how to remove an image assignment from Cloudinary. The procedure is largely identical to import a video from CLoudinary. The differences are explicitly specified at the appropriate place.
+
+1. Click the product in the product list from which you want to remove the Cloudinary image assignment.   
+  The *Edit product* view is displayed.
+
+  ![Edit product](../../Assets/Screenshots/PIM/Products/List/EditProduct.png "[Edit product]")
+
+2. Navigate to the Cloudinary image attribute in the *Attributes* tab.   
+  All assigned images are displayed in the Cloudinary image attribute box.
+
+  ![Cloudinary images](../../Assets/Screenshots/PIM/Products/List/Attributes/CloudinaryImages.png "[Cloudinary images]")
+
+3. Hover over the preview tile of the image to be removed.    
+  The ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete) button is displayed in a small circle in the upper right corner of the preview tile.    
+
+4. Click the ![Delete](../../Assets/Icons/Cross01.png "[Delete]") (Delete) button in the upper right corner of the preview tile.   
+  The *Delete image* window is displayed.   
+
+  ![Delete image](../../Assets/Screenshots/PIM/Products/List/Attributes/CloudinaryDeleteImage.png "[Delete image]")
+
+5. Depending on your objective, click on one of the following buttons displayed in the *Delete image* window:
+
+  - [REMOVE ASSIGNMENT (KEEP IN CLOUDINARY)]   
+  Only the assignment to the image will be removed. When displaying the list of unassigned items, the image may be listed again. The *Delete image* window is closed.
+  - [REMOVE ASSIGNMENT (DELETE IN CLOUDINARY)]   
+  The assignment to the image will be removed and the image itself will be deleted in Cloudinary when saving the product. The *Delete image* window is closed.
+
+    > [Info] Note that problems may occur when deleting an image to which other dependencies exist, for instance if the image is already available in a web shop.
```

Comment:
Vielleicht dazu: Therefore, it is highly recommended not to delete any images from Cloudinary. (o.ä.?)